### ISSUE TITLE
Can I connect to altserver remotely?

### ISSUE THREAD

**Issue by FastBoot-pixellab** [rileytestut/AltStore#1106]
I have a friend who only has an iPhone. He needs to install the IPA, but this is not possible; he does not have a PC. I want to try to install it through my PC but a friend in another city.
Buy an android do not offer money he does not have

**Comment by Rihcus** [comment]
I don't believe so I tried using tailscale (a mesh vpn) with it and it failed to find the server.

**Comment by doccstat** [comment]
Install SideStore either by appdb or some other services. After that he can refresh sidestore and apps without a PC.

**Comment by drakeg77** [comment]
Could you use JIT-streamer and use this method? You would still need a PC at home, but could you use this to remotely sign apps?

**Comment by 12f23eddde** [comment]
Apple's device discovery relys heavily on mDNS (at least in my knowledge), which is out of the scope of Tailscale and other VPN services.

**Comment by TheRealMaN** [comment]
is there any ways to connect remotely to AltServer now?

**Comment by rebelancap** [comment]
Sidestore. Time to abandon AltStore.
